src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -543,18 +556,9 @@ def __eq__(self, other: EnsembleConfig):
         return True
 
     def get_summary_keys(self) -> List[str]:
-        return sorted(self.getKeylistFromImplType(SummaryConfig))
-
-    def get_user_summary_keys(self):
-        return self._user_summary_keys
-
-    def get_node_observation_keys(self, key) -> List[str]:
-        node = self[key]
-        keylist = []
-
-        if isinstance(node, SummaryConfig):
-            keylist = node.get_observation_keys()
-        return keylist
+        if "summary" in self and isinstance(self["summary"], SummaryConfig):
+            return sorted(list(set(self["summary"].keys)))
```

Comment:
Potentially remove `list(set(`. `sorted` already takes a sequence of stuff and turns it into a list.

Reply:
There are duplicates here apparently, so keys `set`, removed explicit cast to list